File: Space Invaders/Assets/Scripts/SpaceInvaders/AudioManager.js
#pragma strict

var killSource: AudioSource;
var shootSource: AudioSource;
var explosionSource: AudioSource;

static var kill : AudioSource;
static var shoot : AudioSource;
static var explosion : AudioSource;

function Start () {
	kill=killSource;
    shoot=shootSource;
    explosion=explosionSource;
}

function Update () {
	
}

static function PlayKill(){
    kill.Play(0);
}

static function PlayShoot(){
    shoot.Play(0);
}

static function PlayExplosion(){
    explosion.Play(0);
}

File: Space Invaders/Assets/Scripts/AudioManager.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class AudioManager : MonoBehaviour
{
    public AudioSource killSource;
    public AudioSource shootSource;
    public AudioSource explosionSource;

    public static AudioSource kill;
    public static AudioSource shoot;
    public static AudioSource explosion;

    void Start()
    {
        kill = killSource;
        shoot = shootSource;
        explosion = explosionSource;
    }
		
    public static void PlayKill()
    {
        kill.Play(0);
    }

    public static void PlayShoot()
    {
        shoot.Play(0);
    }

    public static void PlayExplosion()
    {
        explosion.Play(0);
    }

}
